geo/src/algorithm/is_convex.rs

```diff
@@ -142,6 +144,19 @@ fn is_convex_shaped<T>(
 where
     T: GeoNum,
 {
+    if coords.is_empty() {
```

Comment:
If I comment out this change, all the tests continue to pass. See https://github.com/georust/geo/compare/main...mkirk%2Ffix-hull?body=&expand=1

Can you add a test that exemplifies the bug that this code is fixing?

Reply:
added a test with a degenerate linestring containing a single point